src/ert/gui/simulation/run_dialog.py

```diff
@@ -44,8 +44,9 @@
 class RunDialog(QDialog):
     simulation_done = Signal(bool, str)
 
-    def __init__(self, config_file, run_model, parent=None):
+    def __init__(self, config_file: str, run_model, parent=None):
```

Comment:
Why `run_model` doesn't have type?

Reply:
ah forgot to remove the `: str` addition, sorry, fixed